frontend/src/static/css/product.scss

```diff
@@ -190,4 +191,128 @@
       }
     }
   }
+
+  .form-new-register {
+    display: flex;
+    flex-direction: column;
+    width: 50%;
+    background: #fff;
+    box-shadow: 0px 4px 10px rgba(157, 157, 157, 0.25);
+    border-radius: 14px 0px 0px 14px;
+    border: none;
+    border-right: 1px solid #D7D7D7;
+
+    form {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      width: 100%;
+      height: 100%;
+
+      .input-photo {
+        display: none;
+      }
+      
+      .title {
+        color: #515151;
+        margin: 1% 0;
+        width: 90%;
+        font-size: 14px;
+        font-weight: 500;
+      }
+
+      input {
+        outline: none;
+        width: 90%;
+        height: 5%;
+        padding-left: 1%;
+        margin-bottom: 2%;
+        border: 1px solid #949494;
+        border-radius: 10px;
+        font-size: 16px;
+        color: #333;
+        transition: .24s;
+
+        &:focus {
+          transition: .24s;
+          border: 1px solid #837c7c78;
+        }
+      }
+
+      textarea {
+        outline: none;
+        width: 90%;
+        height: 15%;
+        font-size: 16px;
+        resize: none;
+        color: #333;
+        padding: 1% 1%;
+        margin-bottom: 2%;
+        border-radius: 10px;
+        border: 1px solid #949494;
+        transition: .24s;
+
+        &:focus {
+          transition: .24s;
+          border: 1px solid #837c7c78;
+        }
+      }
+
+      .select-platform {
+        outline: none;
+        width: 91%;
+        margin-bottom: 2%;
+        border-radius: 10px;
+      }
+
+      .content-btn {
+        width: 100%;
+        height: 5%;
+        display: flex;
+        justify-content: flex-end;
+
+        .confirm-btn {
+          background: $main-color
+        }
+
+        .cancel-btn {
+          background: #949494;
+        }
+
+        button {
+          outline: none;
+          cursor: pointer;
+          margin-top: 4%;
+          color: #fff;
+          font-size: 14px;
+          width: 20%;
+          height: 100%;
+          margin-right: 5%;
+          border-radius: 14px;
+          border: none;
+          transition: .24s;
+
+          &:hover {
+            transition: .24s;
+            background: #1d92d6;
+          }
+        }
+      }
+    }
+    .thumbail {
+      cursor: pointer;
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      align-items: center;
+      margin: 3% 0;
+      width: 90%;
+      height: 25%;
```

Comment:
Don't set the height of component fixed, you can break the layout in some screens. When you set the width of a image for example, height is set automatically